tests/src/Functional/SmsFrameworkGatewayAdminTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\Tests\sms\Functional;
+
+use Drupal\Core\Url;
+use Drupal\sms\Entity\SmsGateway;
+
+/**
+ * Tests manage event access page.
+ *
+ * \Drupal\sms\Tests\SmsFrameworkGatewayAdminTest to be migrated to this class.
```

Comment:
> Are we already migrating WebTests to BrowserTests?

I don't really have the desire to do this at this time.

We have nothing to worry about until its [removed](https://www.drupal.org/node/2735005) in Drupal 9.x